src/Backend/Modules/Extensions/Actions/EditThemeTemplate.php

```diff
@@ -346,6 +351,24 @@ private function validateForm(): void
                     $item['active'] = true;
                 }
 
+                $imagePath = FRONTEND_FILES_PATH . '/Templates/images';
+
+                if ($this->record['default_image'] !== null
+                    && ($this->form->getField('remove_default_image')->isChecked()
+                        || $this->form->getField('default_image')->isFilled())
```

Comment:
while the indentation is correct, when you are reading it on first glance it doesn`t look correct
maybe close that second bracket on a new line so it is more clear?

Reply:
Fixed 40e3308c2